src/lex.cpp

```diff
@@ -108,6 +108,8 @@
   case '8':                     \
   case '9'
 
+// TODO: add more space characters.
+
```

Comment:
This comment looks vestigial. Delete?

Reply:
removed it at f1854c0b9168e7f16d85fbf96a8603cb10fff831